FF/FF.C

```diff
@@ -161,6 +161,13 @@ bool preciceAdapter::FF::FluidFluid::addReaders(std::string dataName, Interface*
             new Velocity(mesh_, nameU_));
         DEBUG(adapterInfo("Added reader: Velocity."));
     }
+    else if (dataName.find("U_vol") == 0)
```

Comment:
Is it documented somewhere why we need the additional `U_vol`, while `p_rgh` is just as we take it from OpenFOAM?